media-tv/mythtv/mythtv-30.0.ebuild

```diff
@@ -0,0 +1,408 @@
+	x265? (	media-libs/x265 )
+	xml? ( >=dev-libs/libxml2-2.6.0 )
+	xvid? ( >=media-libs/xvid-1.1.0 )
+	zeroconf? (
+		dev-libs/openssl:0=
+		net-dns/avahi[mdnsresponder-compat]
+	)
+"
+RDEPEND="${COMMON}
+	!media-tv/mythtv-bindings
+	!x11-themes/mythtv-themes
+	media-fonts/corefonts
+	media-fonts/dejavu
+	media-fonts/liberation-fonts
+	x11-apps/xinit
+	autostart? (
+		net-dialup/mingetty
+		x11-apps/xset
+		x11-wm/evilwm
+	)
+	dvd? ( media-libs/libdvdcss )
+	xmltv? ( >=media-tv/xmltv-0.5.43 )
+"
+DEPEND="${COMMON}
+	dev-lang/yasm
+	virtual/pkgconfig
+	x11-base/xorg-proto
+"
+
+S="${WORKDIR}/${PF}/mythtv"
+
+DISABLE_AUTOFORMATTING="yes"
+DOC_CONTENTS="
+To have this machine operate as recording host for MythTV,
+mythbackend must be running. Run the following:
+rc-update add mythbackend default
+
+Your recordings folder must be owned 'mythtv'. e.g.
+chown -R mythtv /var/lib/mythtv
+
+Want mythfrontend to start automatically?
+Set USE=autostart. Details can be found at:
+https://dev.gentoo.org/~cardoe/mythtv/autostart.html
+
+Note that the systemd unit now restarts by default and logs
+to journald via the console at the notice verbosity.
+"
+
+MYTHTV_GROUPS="video,audio,tty,uucp"
+
+pkg_setup() {
+	python-single-r1_pkg_setup
+	enewuser mythtv -1 /bin/bash /home/mythtv ${MYTHTV_GROUPS}
+	usermod -a -G ${MYTHTV_GROUPS} mythtv
+}
+
+src_prepare() {
+	default
+
+	# Perl bits need to go into vender_perl and not site_perl
+	sed -e "s:pure_install:pure_install INSTALLDIRS=vendor:" \
+		-i "${S}"/bindings/perl/Makefile
+
+	# Fix up the version info since we are using the fixes/${PV} branch
+	echo "SOURCE_VERSION=\"v${MY_PV}\"" > "${S}"/VERSION
+	echo "BRANCH=\"${MYTHTV_BRANCH}\"" >> "${S}"/VERSION
+	echo "SOURCE_VERSION=\"${BACKPORTS}\"" > "${S}"/EXPORTED_VERSION
+	echo "BRANCH=\"${MYTHTV_BRANCH}\"" >> "${S}"/EXPORTED_VERSION
+
+	echo "setting.extra -= -ldconfig" >> "${S}"/programs/mythfrontend/mythfrontend.pro
+}
+
+src_configure() {
+	local myconf=
+
+	# Setup paths
+	myconf="${myconf} --prefix=${EPREFIX}/usr"
+	myconf="${myconf} --libdir=${EPREFIX}/usr/$(get_libdir)"
+	myconf="${myconf} --libdir-name=$(get_libdir)"
+	myconf="${myconf} --mandir=${EPREFIX}/usr/share/man"
+
+	# CPU settings
+	for i in $(get-flag march) $(get-flag mcpu) $(get-flag mtune) ; do
+		[ "${i}" = "native" ] && i="host"
+		myconf="${myconf} --cpu=${i}"
+		break
+	done
+	myconf="${myconf} $(use_enable altivec)" # This is only for PPC, not x86.  Who uses this?
+
+	# Input Support
+	myconf="${myconf} $(use_enable input_devices_joystick joystick-menu)"
+	myconf="${myconf} $(use_enable lirc)"
+	myconf="${myconf} $(use_enable cec libcec)"
+	myconf="${myconf} $(use_enable v4l v4l2)"
+	myconf="${myconf} $(use_enable ivtv)"
+	myconf="${myconf} $(use_enable hdpvr)"
+	myconf="${myconf} $(use_enable ieee1394 firewire)"
+	myconf="${myconf} $(use_enable hdhomerun)"
+	myconf="${myconf} $(use_enable vbox)"
+	myconf="${myconf} $(use_enable ceton)"
+	# myconf="${myconf} $(use_enable dveo asi)"
```

Comment:
Just do --disable-foo here, instead of this. Remove the USE flag from IUSE if it has no function in the ebuild. And remove the comments about this below